web_many2one_simple/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Tecnativa - Alexandre D. Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Web Many2One Simple",
+    "summary": """Simple one2many field""",
```

Comment:
```suggestion
    "summary": "Simple many2one widget",
```

Reply:
This one can be the name and without summary

Reply:
So.. better rename the whole module to "web_widget_many2one_simple" ?

Reply:
Well, I didn't mean that, but removing the summary key in the manifest and putting this as name key, but you are totally right that the name of the module is not correct.